datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/mutate/util/OwnerUtils.java

```diff
@@ -107,37 +109,21 @@ public static MetadataChangeProposal buildRemoveOwnersProposal(
                 Constants.OWNERSHIP_ASPECT_NAME,
                 entityService,
                 new Ownership());
+    assert ownershipAspect != null;
```

Comment:
Never use asserts, this results in a Java Error (intended for fatal flows) and will produce undefined behavior.